File: C#/URI/1046.cs
using System; 

class URI {

    static void Main(string[] args) { 

         int total;
        string[] linha = Console.ReadLine().Split(' ');

        int a = int.Parse(linha[0]);
        int b = int.Parse(linha[1]);

        if (a == b)
            Console.WriteLine("O JOGO DUROU 24 HORA(S)");
        else if (a > b)
        {
            total = a - 24;
            total = Math.Abs(total) + b;
            Console.WriteLine("O JOGO DUROU {0} HORA(S)", total);
        }
        else
        {
            total = b - a;
            Console.WriteLine("O JOGO DUROU {0} HORA(S)", total);
        }

    }

}

File: Python/URI/1046.py
linha = input().split(' ')

a = int(linha[0])
b = int(linha[1])

if(a == b):
    print("O JOGO DUROU 24 HORA(S)")
elif(a > b):
    total = a - 24
    total = abs(total) + b
    print ("O JOGO DUROU " +str(total)+" HORA(S)"); 
else:
    total = b-a
    print("O JOGO DUROU " +str(total)+" HORA(S)")